src/GitHub.App/Controllers/UIController.cs

```diff
@@ -421,8 +421,7 @@ void ResetViewModel(UIViewType viewType)
             if (flowList.TryGetValue(viewType, out pair))
             {
                 var vm = pair.ViewModel as IResettable;
-                //TODO:Can I even do this?
-                //vm?.Reset();
+                vm?.Reset();
```

Comment:
This seems like a handy C# 6 thing! What I'm doing here is using the null conditional operator to  speculate that the VM could be a member of `IResettable` and to access `Reset()` if it is, so if `vm` is null, and the operator avoids a null reference exception